guide/de/ch_invest.xml

```diff
@@ -859,13 +859,13 @@ Einnahmen
                 </listitem>
 
                 <listitem>
-                  <para><guilabel>WKN, ISIN oder anderer Code</guilabel> gibt an, wo Sie andere Codenummern oder -text
+                  <para><guilabel>WKN, ISIN oder anderer Code</guilabel> gibt an, wo Sie andere Codenummern oder -texte
```

Comment:
Gibt es was besseres als Codenummern oder -texte?
Vielleicht `alphanumerisches Kürzel `. Dann vielleicht auch Kürzel statt Code in die Bezeichnung im Programm (de.po) übernehmen.